css/src/components/popover.scss

```diff
@@ -0,0 +1,40 @@
+$popover-background-color: $body-background !default;
+$popover-border: $border-width solid $border-white-high-contrast !default;
+$popover-border-radius: $border-radius !default;
+$popover-shadow: $box-shadow-heavy !default;
+$popover-width: 224px !default;
+
+.popover {
+	display: inline-block;
+	position: relative;
+
+	.popover-summary {
```

Comment:
I think we should remove the need for this class and use `summary`. We need to take a hard dependency on the details element's `open` attribute so requiring `popover-summary` to reset the default summary styles or get the dismiss pseudo element seems like it complicates usage unnecessarily.

Reply:
updated